mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

```diff
@@ -140,15 +143,15 @@ def select(self, query: ast.Select) -> pd.DataFrame:
         search_params = {"ids": []}
         subset_where_conditions = []
         for op, arg1, arg2 in where_conditions:
-            if arg1 == 'ids':
+            if arg1 == 'id':
                 if op == '=':
-                    search_params["ids"] = arg2
+                    search_params["ids"].append(arg2)
```

Comment:
This logic which stores the id in the query as `ids` should also be changed; it is unnecessary and unclean.

Reply:
i have made the change of it, please review again